ci/run.sh

```diff
@@ -63,3 +63,18 @@ if [ "$machine" != "MinGw" ]; then
     run_function $path $function_name $image "$create_args" $input_data $expected_data $runtime
   done
 fi
+
+# run application
+source `dirname "${BASH_SOURCE[0]}"`/../applications/helpers.sh
+
+for test in java-boot; do
+  path=`dirname "${BASH_SOURCE[0]}"`/../applications/${test}
+  application_name=fats-application-uppercase-${test}
+  image=$(fats_image_repo ${application_name})
```

Comment:
this image name will collide with the function build for `java-boot`, while the image should be deleted, it's a good practice to use unique names to avoid potential conflicts, like a registry that's slow to delete an image.

Reply:
The image here has `application` in its name, so I don't think there will be conflict.

Reply:
the `application` token is replace the `cluster` `local` equivalent for functions. It should be additive and we should also test local application builds.

Reply:
filed: https://github.com/projectriff/fats/issues/191 for local builds.